src/Wallabag/CoreBundle/Helper/Redirect.php

```diff
@@ -24,6 +27,10 @@ public function __construct(Router $router)
      */
     public function to($url, $fallback = '')
     {
+        if ($this->actionMarkAsRead == 0) {
```

Comment:
Maybe we should define some constant in Config class to avoid using this not relevant `0`.
Also, use `===` instead of simple `==` and use yoda condition (put the defined value first).